openff/benchmark/torsiondrives/compute.py

```diff
@@ -0,0 +1,380 @@
+
+                        # perform optimization
+                        input_data = self._generate_optimization_input(qcmol_s, compute_spec)
+                        result = self._execute_qcengine(input_data,
+                                                        local_options=local_options,
+                                                        scf_maxiter=scf_maxiter,
+                                                        geometric_maxiter=geometric_maxiter,
+                                                        geometric_coordsys=geometric_coordsys,
+                                                        geometric_qccnv=geometric_qccnv)
+
+                        # TODO: consider if we need to do multiple optimizations per grid point to
+                        # get robust results?
+                        task_results[gridpoint].append((result.initial_molecule.geometry.flatten().tolist(),
+                                                        result.final_molecule.geometry.flatten().tolist(),
+                                                        result.energies[-1]))
+
+                        opts[spec_name][gridpoint].append(result)
+
+                td_api.update_state(state, task_results)
+
+        # when complete, return output as a JSON-serializable blob
+        return opts
+
+
+    def _execute_qcengine(
+            self, input_data, procedure="geometric", local_options=None,
+            scf_maxiter=None, geometric_maxiter=None, geometric_coordsys=None,
+            geometric_qccnv=None):
+        import qcengine
+
+        # inject reset=True fix, set coordsys to 'dlc'
+        input_data['keywords']['reset'] = True
+        input_data['keywords']['coordsys'] = 'dlc'
+
+        # inject exposed convergence parameters, if specified
+        if scf_maxiter is not None:
+            input_data['input_specification']['keywords']['maxiter'] = scf_maxiter
+        if geometric_maxiter is not None:
+            input_data['keywords']['maxiter'] = geometric_maxiter
+        if geometric_coordsys is not None:
+            input_data['keywords']['coordsys'] = geometric_coordsys
+        if geometric_qccnv is not None:
+            input_data['keywords']['qccnv'] = geometric_qccnv
+
+        return qcengine.compute_procedure(
+                input_data, procedure=procedure, local_options=local_options)
+
+    @staticmethod
+    def _generate_optimization_input(qcmol, compute_spec):
+        import ast
+        from qcelemental.models import OptimizationInput
+
+        method = compute_spec['method']
+        basis = compute_spec['basis']
+        program = compute_spec['program']
+
+        # generate optimization inputs
+        input_data = OptimizationInput(
+                keywords={
+                      "coordsys": "dlc",
+                      "enforce": 0,
+                      "epsilon": 1e-05,
+                      "reset": True,
+                      "qccnv": False,
```

Comment:
By default torsiondrive uses `qccnv=True` so we want this to be our default as well I think.

Reply:
Good eye, will set this.